data_model/src/events/mod.rs

```diff
@@ -88,7 +88,9 @@ pub enum EventSubscriberMessage {
 }
 
 /// Event.
-#[derive(Debug, Clone, PartialEq, Eq, Decode, Encode, FromVariant, IntoSchema)]
```

Comment:
`Serialize` and `Deserialize` was missing from events. I added it, but I wonder if it was on purpose?

Reply:
We might need the two soon